lib/diary.js

```diff
@@ -0,0 +1,141 @@
+      var caloriesLeft = (goalBreakfast - totalBreakfast)
+      $("table.left > tfoot > tr#goal").append("<td>" + caloriesLeft + "</td>")
+      $("table.left > tfoot > tr#goal > td").addClass('green');
+      $("table.left > tfoot > tr#goal > td:contains('-')").addClass('red');
+    })
+    .catch(function(error){
+      console.error(error)
+    })
+  $.get("https://hidden-shelf-49347.herokuapp.com/api/v1/meals")
+    .then(function(foods){
+      $("h2.right").append(foods[1].name)
+      $("div.right").append($table1.addClass("right"))
+      var totalSnack = 0
+      var goalSnack = 200
+      foods[1].foods.forEach(function(element) {
+        $("table.right > tbody").append("<tr><td>" + element.name + "</td><td>" + element.calories + "</td></tr>")
+        totalSnack += element.calories
+      })
+      $("table.right > tfoot > tr#total").append("<td>" + totalSnack + "</td>")
+      $("table.right > tfoot > tr#goal").append("<td>" + (goalSnack - totalSnack) + "</td>")
+      $("table.right > tfoot > tr#goal > td").addClass('green');
+      $("table.right > tfoot > tr#goal > td:contains('-')").addClass('red');
+    })
+    .catch(function(error){
+      console.error(error)
+    })
+  $.get("https://hidden-shelf-49347.herokuapp.com/api/v1/meals")
+    .then(function(foods){
+      $("h2.left1").append(foods[2].name)
+      $("div.left1").append($table2.addClass("left1"))
+      var totalLunch = 0
+      var goalLunch = 600
+      foods[2].foods.forEach(function(element) {
+        $("table.left1 > tbody").append("<tr><td>" + element.name + "</td><td>" + element.calories + "</td></tr>")
+        totalLunch += element.calories
+      })
+      $("table.left1 > tfoot > tr#total").append("<td>" + totalLunch + "</td>")
+      $("table.left1 > tfoot > tr#goal").append("<td>" + (goalLunch - totalLunch) + "</td>")
+      $("table.left1 > tfoot > tr#goal > td").addClass('green');
+      $("table.left1 > tfoot > tr#goal > td:contains('-')").addClass('red');
+    })
+    .catch(function(error){
+      console.error(error)
+    })
+  $.get("https://hidden-shelf-49347.herokuapp.com/api/v1/meals")
+    .then(function(foods){
+      $("h2.right1").append(foods[3].name)
+      $("div.right1").append($table3.addClass("right1"))
+      var totalDinner = 0
+      var goalDinner = 800
+      foods[3].foods.forEach(function(element) {
+        $("table.right1 > tbody").append("<tr><td>" + element.name + "</td><td>" + element.calories + "</td></tr>")
+        totalDinner += element.calories
+      })
+      $("table.right1 > tfoot > tr#total").append("<td>" + totalDinner + "</td>")
+      $("table.right1 > tfoot > tr#goal").append("<td>" + (goalDinner - totalDinner) + "</td>")
+      $("table.right1 > tfoot > tr#goal > td").addClass('green');
+      $("table.right1 > tfoot > tr#goal > td:contains('-')").addClass('red');
+    })
+    .catch(function(error){
+      console.error(error)
+    })
+})
+var $table = $('<table/>').append(
```

Comment:
Tables work fine as is and this would be a potential refactor, but I'm wondering, what was the need for dynamically creating 4 separate table variables?